rlclientlib/utility/stl_container_adapter.h

```diff
@@ -0,0 +1,27 @@
+﻿#pragma once
+#include "data_buffer.h"
+#include <iterator>
+
+namespace reinforcement_learning {namespace utility {
+  class data_buffer;
+  class stl_container_adapter {
```

Comment:
forward declaration not necessary if "data_buffer.h" is included.

Reply:
Makes sense.